src/data/dataMethods.ts

```diff
@@ -323,6 +323,10 @@ export function _requestObservable<R>(
     if (typeof perspective === 'string' && perspective !== 'raw') {
       validateApiPerspective(perspective)
       options.query = {perspective, ...options.query}
+      // If the perspective is set to `previewDrafts` we can't use the CDN, the API will throw
+      if (perspective === 'previewDrafts') {
+        useCdn = false
+      }
```

Comment:
Should we warn when this happens?

E.g. something like
```suggestion
      if (useCdn && perspective === 'previewDrafts') {
        useCdn = false
        console.warn(`perspective: 'previewDrafts' isn't supported with useCdn: true`)
      }
```

Reply:
I think we should. We have a warning printer which ensures we only print once - should probably use that?
https://github.com/sanity-io/client/blob/main/src/warnings.ts#L5